packages/platform-sdk-profiles/src/drivers/memory/contacts/contact-address.ts

```diff
@@ -136,6 +136,8 @@ export class ContactAddress implements IContactAddress {
 		try {
 			this.#wallet = await this.#coin.client().wallet(this.address());
 		} catch {
+			// @TODO: remove this in a refactor
```

Comment:
This will be addressed in https://github.com/ArkEcosystem/platform-sdk/pull/1399